src/__test__/graphql-flow.test.js

```diff
@@ -54,6 +54,32 @@ describe('graphql-flow generation', () => {
         `);
     });
 
+    it('should split types', () => {
+        const result = rawQueryToFlowTypes(
+            `
+            query SomeQuery($id: String!) {
+                human(id: $id) { id }
+            }
+        `,
+            {splitTypes: true},
+        );
+
+        expect(result).toMatchInlineSnapshot(`
+            export type SomeQueryType = {|
+                variables: {|
+              id: string
+            |},
+                response: {|
+
+              /** A human character*/
+              human: ?{|
+                id: string
+              |}
+            |}
+            |};
+        `);
```

Comment:
Is this an issue with jest's snapshotting or is the indentation actually a bit wonky?  No action req'd, just curious.

Reply:
jest snapshotting 😭